components/commands/common/accelerator_parsing.cc

```diff
@@ -155,6 +147,7 @@ std::string ToCodesString(const ui::Accelerator& accelerator) {
 }
 
 ui::Accelerator FromCodesString(const std::string& value) {
+  CHECK(!value.empty());
```

Comment:
This `CHECK` also seems dangerous - we could have accidentally written an empty string into the prefs file before, and now the users browser won't launch.

I guess because we have the `CHECK` in the `accelerator_pref_manager` this is safe but it still makes me a little nervous :laughing: